test/augmentation/test_augmentation_3d.py

```diff
@@ -0,0 +1,748 @@
+from typing import Union, Tuple
+
+import pytest
+import torch
+import torch.nn as nn
+
+from torch.testing import assert_allclose
+from torch.autograd import gradcheck
+
+import kornia
+import kornia.testing as utils  # test utils
+from kornia.constants import pi
+from kornia.augmentation import (
+    RandomDepthicalFlip3D,
+    RandomHorizontalFlip3D,
+    RandomVerticalFlip3D,
+    RandomAffine3D,
+    RandomRotation3D
+)
+
+
+class TestRandomHorizontalFlip3D:
+
+    def smoke_test(self, device):
+        f = RandomHorizontalFlip3D(0.5)
+        repr = "RandomHorizontalFlip3D(p=0.5, return_transform=False)"
+        assert str(f) == repr
+
+    def test_random_hflip(self, device):
+
+        f = RandomHorizontalFlip3D(p=1.0, return_transform=True)
+        f1 = RandomHorizontalFlip3D(p=0., return_transform=True)
+        f2 = RandomHorizontalFlip3D(p=1.)
+        f3 = RandomHorizontalFlip3D(p=0.)
+
+        input = torch.tensor([[[0., 0., 0., 0.],
+                               [0., 0., 0., 0.],
+                               [0., 0., 1., 2.]]])  # 1 x 3 x 4
```

Comment:
BTW, I think this is wrong. The flip is happening in the vertical axis

Reply:
What is the definition here? I used:
- Horizontal: flip(-1)
- Vertical: flip(-2)
- Depthical: flip(-3)

Reply:
ok, then is fine I guess